packages/kbn-investigation-shared/src/schema/investigation_item.ts

```diff
@@ -8,11 +8,16 @@
 
 import * as t from 'io-ts';
 
-const itemSchema = t.type({
-  title: t.string,
-  type: t.string,
-  params: t.record(t.string, t.any),
-});
+const itemSchema = t.intersection([
+  t.type({
+    title: t.string,
+    type: t.string,
+    params: t.record(t.string, t.any),
+  }),
+  t.partial({
+    id: t.string,
+  }),
+]);
```

Comment:
As said earlier, I think we don't have to change anything here. If the embbedable panel system needs an id returned, let's give them one. If we need to store it, let's store it under `params.xxxId`.

Reply:
Yep that's a leftover. I'll clean it up.